docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_arm_template_ado/_index.md

```diff
@@ -111,52 +111,47 @@ In this scenario, you will create a new Release pipeline to deploy the environme
 
     ![Screenshot of ARM template config](./11.jpg)
 
+  * *sshRSAPublicKey* - Your ssh public key
+  * *spnClientId* - Your Azure service principal name
+  * *spnClientSecret* - Your Azure service principal password
+  * *spnTenantId* - Your Azure tenant ID
+  * *windowsAdminUsername* - Client Windows VM admin username
+  * *windowsAdminPassword* - Client Windows VM admin password
+  * *myIpAddress* - Public IP address of your network
+  * *logAnalyticsWorkspaceName* - Unique Log Analytics workspace name
+  * *deploySQLMI* - SQL Managed Instance deployment (true/false)
+  * *SQLMIHA* - SQL Managed Instance high-availability deployment (true/false)
+  * *deployPostgreSQL* - PostgreSQL Hyperscale deployment (true/false)
   * *clusterName* - AKS cluster name
+  * *bastionHostName* - Indicate whether to deploy bastion host to manage AKS
   * *dnsPrefix* - AKS unique DNS prefix
-  * *nodeAdminUsername* - AKS Node Username
-  * *sshRSAPublicKey* - Your ssh public key
-  * *SPN_CLIENT_ID* - Your Azure service principal name
-  * *SPN_CLIENT_SECRET* - Your Azure service principal password
-  * *SPN_TENANT_ID* - Your Azure tenant ID
-  * *ARC_DC_SUBSCRIPTION* - Azure Arc Data Controller Azure subscription ID
-  * *ARC_DC_REGION* - Azure location where the Azure Arc Data Controller resource will be created in Azure (Currently, supported regions supported are eastus, eastus2, centralus, westus2, westeurope, southeastasia)  
-  * *SPN_AUTHORITY* - *https://login.microsoftonline.com* **Do not change**
   * *kubernetesVersion* - AKS Kubernetes Version (See previous prerequisite)
-  * *adminUsername* - Client Windows VM admin username
-  * *adminPassword* - Client Windows VM admin password
-  * *vmSize* - Client Windows VM size
-  * *resourceGroup* - Azure resource group where all the resources get deploy
-  * *AZDATA_USERNAME* - Azure Arc Data Controller admin username.  DO NOT USE 'sa' or 'admin'!!
-  * *AZDATA_PASSWORD* - Azure Arc Data Controller admin password (The password must be at least 8 characters long and contain characters from three of the following four sets: uppercase letters, lowercase letters, numbers, and symbols.)
-  * *ACCEPT_EULA* - "yes" **Do not change**
-  * *ARC_DC_NAME* - Azure Arc Data Controller name. The name must consist of lowercase alphanumeric characters or '-', and must start d end with a alphanumeric character (This name will be used for k8s namespace as well).
-  * *MSSQL_MI_NAME* - SQL Managed Instance name to be deployed on the Kubernetes cluster
-
+  * *githubAccount* - Target GitHub account
+  * *githubBranch* - Target GitHub branch
+  
     > **Note: Make sure that you are using the same Azure resource group name as the one you've just used in the *azuredeploy.parameters.json* file**
 
     ![Screenshot of ARM template parameters config](./12.jpg)
 
     ![Screenshot of ARM template parameters config](./13.jpg)
 
-    ![Screenshot of ARM template parameters config](./14.jpg)
```

Comment:
You have a file jump here. All images should be aligned and in the right order. i.e 13,14,15...

Reply:
Updated